### DISCUSSION TITLE
参考文献链接不全

### DISCUSSION THREAD

**Discussion by Shaun-Gin** [nju-lug/NJUThesis#91] [Q&A]
参考文献链接随换行断开，无法正常访问，如图：
<img width="807" alt="Screen Shot 2022-03-07 at 7 01 25 PM" src="https://user-images.githubusercontent.com/44497970/157021872-91f24e67-eee0-4909-b36c-27da90d3abdf.png">

不使用模板则正常，如图：
<img width="705" alt="Screen Shot 2022-03-07 at 6 54 47 PM" src="https://user-images.githubusercontent.com/44497970/157021936-55924024-1956-4cae-93c6-bd06b3366dc3.png">
bib：

@misc{pressure,
	date-added = {2022-03-04 16:24:46 +0800},
	date-modified = {2022-03-07 17:52:44 +0800},
	title = {Pressure conversion multiplication chart},
	url = {https://www.sigmaaldrich.com/US/en/technical-documents/technical-article/chemistry-and-synthesis/organic-reaction-toolbox/pressure-conversion}}

**Comment by atxy-blip** [comment]
呃 这个看上去是模板 bug，应该发到 issue 区。我转移过去吧。

-----

刚刚尝试复现，并没出现你描述的错误，因此没转移 issue。
*请提供你的编译环境信息*。

测试情况：

在我的机子上没有产生问题（Windows, TeX Live 2021，模板版本为仓库内最新）：

![image](https://user-images.githubusercontent.com/63113546/157029008-1881fa64-5c05-470f-96c6-8a7a6af2e3a8.png)

在 <tex.nju.edu.cn> 没有产生问题（模板版本为 v0.15.0）：

![image](https://user-images.githubusercontent.com/63113546/157029273-1dd3e34f-0155-4cf5-a31c-7b604f4b7c81.png)

**Reply by Shaun-Gin** [reply]
是我没有关闭草稿模式的原因，关闭之后正常了
